caffeine/src/main/java/com/github/benmanes/caffeine/cache/AbstractLinkedDeque.java

```diff
@@ -440,7 +440,7 @@ public void remove() {
      * If the expected modCount value that the iterator believes that the backing deque should have
      * is violated then the iterator has detected concurrent modification.
      */
-    void checkForComodification() {
+    void checkForConcurrentModification() {
```

Comment:
This word is too close to Commodification which is something else, and as this is not a public function, it seemed reasonable to spell it out (java isn't particularly shy about long method names...)